### DISCUSSION TITLE
Version Number

### DISCUSSION THREAD

**Discussion by WaielHakbah** [spatie/laravel-settings#186] [Q&A]
Hi, 
I see in the change log the version number of the latest release is 1.6.1.  
Also in the release it's 1.6.1 did the version number change for the package ? 
last version is being installed through composer is 2.6.0,  should it be 2.6.1 ?

**Comment by rubenvanassche** [comment]
Yeah this was a mistake of mine, I've corrected it. We stopped developing v1.

**Comment by Hum4n01d** [comment]
Just got an email from GitHub that I was mentioned in this thread?

**Reply by RubenGres** [reply]
Same for me, I never even heard of this repository before

**Reply by Hum4n01d** [reply]
Nope, me neither!

**Reply by rubenvanassche** [reply]
Guess something is going wrong at GitHub, because we haven't mentioned or added you in as a participant 😄

**Reply by arfizato** [reply]
Same here haha

**Reply by moy** [reply]
A spam was posted mentioning many users (including me), and then deleted but we're still considered as part of the discussion apparently.
